telemetry/definitions/commonDefinitions.json

```
@@ -773,7 +773,44 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "authConnectionsCount",
+            "type": "int",
+            "description": "Current number of authentication connections the user has."
+        },
+        {
+            "name": "authUIElement",
+            "type": "string",
+            "description": "Feature tab user is interacting in.",
+            "allowedValues": [
+                "awsResource",
+                "codewhisperer",
+                "codecatalyst"
+            ]
+        },
+        {
+            "name": "unlockedTabs",
+            "type": "string",
+            "description": "comma delimtied list of tabs that user already has valid connections to."
+        },
+        {
+            "name": "invalidInputFields",
+            "type": "string",
+            "description": "comma delimtied list of input fields that user has invalid inputs typed."
+
+        },
+        {
+            "name": "region",
+            "type": "string",
+            "description": "Region selected by user." 
```

Generic names should probably have generic (but precise) descriptions:
```suggestion
            "description": "An AWS region id (e.g. `us-east-1) associated with the event" 
```

VS Code has also been using `awsRegion` for a while now. Might make sense to use the same name here even if it's a little redundant.